.travis.yml

```diff
@@ -2,7 +2,7 @@
 language: node_js
 node_js: 8
 sudo: false
-script: npm run build
+script: npm run build && npm run eslint
```

Comment:
* Splitting this to two jobs would reduce the feedback time.
* Having a separate package.json would massively speed up the lint test.
* As a simple improvement we should swap the two tests